### PR TITLE
Add npm loglevel environment variable

### PR DESCRIPTION
**Pull request by Starefossen**
This PR reverts #36 and adds `NPM_CONFIG_LOGLEVEL` environment variable according to discussion in docker-library/official-images#543.

### CODE REVIEW

File: 1.5/Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ FROM buildpack-deps:jessie
 RUN gpg --keyserver pool.sks-keyservers.net --recv-keys 9554F04D7259F04124DE6B476D5A82AC7E37093B DD8F2338BAE7501E3DD5AC78C273792F7D83545D
 
 ENV IOJS_VERSION 1.5.1
+ENV NPM_CONFIG_LOGLEVEL info
```
Comment (tianon) [inline 1.5/Dockerfile:6]: If you move this up a few lines, the layer can be cached between version bumps. :smile:

Comment (Starefossen) [inline 1.5/Dockerfile:6]: Nice catch!

**Comment by pesho** [comment]
LGTM :+1:

**Comment by retrohacker** [comment]
LGTM as well

**Pull request merged**